packages/zowe-explorer/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### Bug fixes
 
+- Fixed an issue where Zowe Explorer displayed a "No Zowe client configurations" prompt when a project, user-level config exists but no global config was present. [#3168](https://github.com/zowe/zowe-explorer-vscode/issues/3168)
+- Fixed an issue where the `ProfilesUtils.getProfileInfo` function returned a new `ProfileInfo` instance that did not respect the `ZOWE_CLI_HOME` environment variable and workspace paths. [#3168](https://github.com/zowe/zowe-explorer-vscode/issues/3168)
```

Comment:
what do we mean by "did not respect"? Did the instance simply ignore the variable and workspace path? If so, can we use "ignore" instead?

Reply:
That's correct - updated this entry as well